exasol_script_languages_container_ci_setup/templates/build_buildspec.yaml

```diff
@@ -0,0 +1,36 @@
+version: 0.2
+
+env:
+  shell: bash
+  secrets-manager:
+    DOCKER_USER: "Dockerhub:User"
+    DOCKER_PASSWORD: "Dockerhub:AccessToken"
+phases:
+
+  install:
+    runtime-versions:
+      python: 3.8
+    commands:
+      - git submodule update --init --recursive
+      - pip3 install --upgrade pip
+      - pip3 install "{{ script_languages_ci_location }}"
```

Comment:
This should get installed by the flavor repo

Reply:
Right. Done